src/block/VanillaBlocks.php

```diff
@@ -694,6 +693,7 @@ protected static function setup() : void{
 		self::register("cut_sandstone", $factory->get(24, 2));
 		self::register("cut_sandstone_slab", $factory->get(421, 3));
 		self::register("cyan_glazed_terracotta", $factory->get(229, 2));
+		self::register("campfire", $factory->get(464, 0));
```

Comment:
This is still in the wrong place. It should be between `cake` and `carpet`.

If you're using PhpStorm, select the lines, go Edit -> Sort Lines if you don't know how to sort alphabetically.